homeassistant/components/ruuvitag_ble/sensor.py

```diff
@@ -0,0 +1,165 @@
+"""Support for RuuviTag sensors."""
+from __future__ import annotations
+
+from typing import Optional, Union
+
+from sensor_state_data import (
+    DeviceKey,
+    SensorDescription,
+    SensorDeviceClass,
+    SensorDeviceInfo,
+    SensorUpdate,
+    Units,
+)
+
+from homeassistant import config_entries, const
+from homeassistant.components.bluetooth.passive_update_processor import (
+    PassiveBluetoothDataProcessor,
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothEntityKey,
+    PassiveBluetoothProcessorCoordinator,
+    PassiveBluetoothProcessorEntity,
+)
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+SENSOR_DESCRIPTIONS = {
+    (SensorDeviceClass.TEMPERATURE, Units.TEMP_CELSIUS): SensorEntityDescription(
+        key=f"{SensorDeviceClass.TEMPERATURE}_{Units.TEMP_CELSIUS}",
+        device_class=SensorDeviceClass.TEMPERATURE,
```

Comment:
This is using the wrong class. We should use the Home Assistant sensor integration device class enum, not a 3rd party name.

https://github.com/home-assistant/core/blob/4b89d087bb62dd1da8f5a3728619c670df7c9b90/homeassistant/components/sensor/__init__.py#L88

The description key item member should be the 3rd party instance, but the entity description attribute should be the Home Assistant instance.